app/data/ukis_main.json

```diff
@@ -0,0 +1,5112 @@
+{
```

Comment:
For consistency with the other surveys it should be `1_ukis.json`. Removing the `eq_id` is something we're looking at outside of this pull request.